packages/manager/modules/nutanix/src/onboarding/constants.js

```diff
@@ -0,0 +1,3 @@
+export const GUIDES = [];
```

Comment:
Empty ?

Reply:
this is auto generated code, some data will be added as we continue to work on this project.